lib/resources/example_json/r4_examples_condition.rb

```diff
@@ -102,171 +102,168 @@ module Resources
     }.freeze
 
     R4_PATIENT_CONDITION_ENTRY ||= {
-      'resourceType': 'Condition',
-      'id': 'p109117485',
-      'meta': {
-        'versionId': '109117485',
-        'lastUpdated': '2020-06-11T04:05:04.000Z',
-        'security': [
-          {
-            'system': 'http://terminology.hl7.org/CodeSystem/v3-ActCode',
-            'code': 'PHY',
-            'display': 'physician requested information sensitivity',
-            'userSelected': false
-          },
-          {
-            'system': 'http://terminology.hl7.org/CodeSystem/v3-ActCode',
-            'code': 'PRS',
-            'display': 'patient requested information sensitivity',
-            'userSelected': false
-          }
-        ]
```

Comment:
Why was this removed ? 
Condition has in meta security labels.

Reply:
Can this be put back since we are now supporting the meta.security field?

Reply:
I have added the security label from the previous example into this one